src/AppBundle/Controller/PagesController.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use Doctrine\Common\Collections\Criteria;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+
+class PagesController extends Controller
```

Comment:
all methods in one controller, maybe replace actions to different controllers such as UserController or TagContoller